### PR TITLE
Fixed donation URL

### PR DESCRIPTION
**Pull request by RemDelaporteMathurin**
Hi all, just found a broken link!

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -33,6 +33,6 @@ ranging from laptops to high-performance computers.
 FEniCS is a [NumFOCUS](https://www.numfocus.org/) fiscally supported
 project. If you like FEniCS and want to support our mission to produce
 the best possible platform for open-source computing, consider making [a
-donation](https://numfocus.salsalabs.org/donate-to-fenics/index.html) to our project.
```
Comment (francesco-ballarin) [inline index.md:36]: Simply change this to
```
donation](https://numfocus.org/donate-to-fenics) to our project.
```

**Review by francesco-ballarin — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (RemDelaporteMathurin) [reply index.md:36]: oops must've gotten mixed up in my copy pastes.... thanks

**Review by francesco-ballarin — APPROVED** [review]

**Comment by francesco-ballarin** [comment]
Thanks @RemDelaporteMathurin !

**Comment by francesco-ballarin** [comment]
@RemDelaporteMathurin there seems to be another Donate link in the top right menu, do you mind making another PR to fix that as well? (while you are at it, if possible search if there are further occurrences of the broken url elsewhere). Thanks!

**Pull request merged**